hudi-client/src/main/java/org/apache/hudi/metrics/Metrics.java

```diff
@@ -47,7 +48,7 @@ private Metrics(HoodieWriteConfig metricConfig) {
     if (reporter == null) {
       throw new RuntimeException("Cannot initialize Reporter.");
     }
-    // reporter.start();
+    reporter.start();
```

Comment:
@vinothchandar I noticed that `reporter.start` was annotated before, is there any other concern? IMHO, call `start` method here is reasonable.

Reply:
you mean commented? can you try git blaming the lines and see if there are any clues.. IIRC, there was some issue that made us do this? I am not tbh .. uncommenting looks reasonable to me, but better to fully understand the context